.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules/
 test/test.css
+package-lock.json
```

Comment:
hmm, don't think we want this here

Reply:
@dlockhart I added this because we seem to have it in all the other BrightspaceUI repos and otherwise I was in danger of commiting the package-lock.json which we don't seem to do any where else? Is there a reason why `colors` is different?

Reply:
Oh sorry my bad, I thought this was `package-lock.json` getting _added_, not getting ignored. Carry on...